native/src/test/scala/sqs4s/internal/aws4/aws4suite/CanonicalRequestSpec.scala

```diff
@@ -0,0 +1,408 @@
+package sqs4s.internal.aws4.aws4suite
+
+import cats.effect.IO
+import org.http4s.Method._
+import org.http4s._
+import org.http4s.client.dsl.io._
+import sqs4s.internal.aws4
+import sqs4s.internal.aws4.IOSpec
+import sqs4s.internal.models.CReq
+
+import scala.io.Source
+
+class CanonicalRequestSpec extends IOSpec {
+
+  import aws4.common._
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)